libvast/src/systemd.cpp

```diff
@@ -15,6 +17,27 @@ namespace vast::systemd {
 
 #if VAST_LINUX
 
+bool connected_to_journal() {
```

Comment:
Should this return a `caf::error`? After all, there are many ways this can go wrong.

Reply:
We discussed this; not necessary.